spec/models/consent_form_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+RSpec.describe ConsentForm, type: :model do
+  describe "Validations" do
+    let(:cf) { build(:consent_form) }
+
+    context "when form_step is :name" do
+      before { cf.form_step = :name }
+
+      it do
```

Comment:
Missing description here. Or are these expressible as as a couple one-liners?

Reply:
Actually, these would work well as one-liners ...

```
    subject(:cf) { build(:consent_form) }

    context "when form_step is :name" do
      before { cf.form_step = :name }

      it { should validate_presence_of(:first_name).on(:update) }
      it { should validate_presence_of(:last_name).on(:update) }
    end
```

Reply:
I wrote it like that but one of the linters re-wrote it, lol...